src/tests/ftest/lustre/llmount.yaml

```diff
@@ -0,0 +1,72 @@
+# change host names to your reserved nodes, the
+# required quantity is indicated by the placeholders
+hosts: !mux
```

Comment:
as is you do not need the !mux, which tells avocado to run the test multiple times using different sets of parameters but you just have 1 parameter here.